Changelog.md

```diff
@@ -1,5 +1,69 @@
 ## Ignition Gazebo 6.x
 
+### Gazebo Sim 6.17.0 (2025-01-10)
+
+1. **Baseline:** this includes all changes from 6.16.0 and earlier.
```

Comment:
We probably don't need this line for minor/patch releases, but not a blocker.

Reply:
+1